### PR TITLE
Status update for Unmaintained projects

### PR DESCRIPTION
**Pull request by KiruthikaJanakiraman**
Added a new status 'Unmaintained' for tests in old projects that do not have any commits in the past 2 years.
Used [repo-info](https://github.com/TestingResearchIllinois/idoft/tree/main/repo-info) script to identify old projects.

### CODE REVIEW

**Comment by zzjas** [comment]
Thanks! Can you also verify and mark `https://github.com/odrotbohm/sos,d4ed68a143b637c7e2d523edad1b105bf6c2d996,20-modulith,example.sos.modulith.orders.EmailNotificationTest.completingAnOrderUpdatesInventory,NOD,,,` as `Unmaintained`?

**Comment by KiruthikaJanakiraman** [comment]
Done

**Comment by KiruthikaJanakiraman** [comment]
> What's the date in `Notes`? I don't see it described in README.

The date is the last commit date. I have added the same to README.

**Comment by darko-marinov** [comment]
Can you please: (1) add a script you used to find these tests, (2) change the script to have in the `Notes` something like `last commit on <DATE>`, and (3) squash your changes in 1 commit?

**Comment by KiruthikaJanakiraman** [comment]
> Can you please: (1) add a script you used to find these tests, (2) change the script to have in the `Notes` something like `last commit on <DATE>`, and (3) squash your changes in 1 commit?

I used an existing script in IDoFT: [repo-info](https://github.com/TestingResearchIllinois/idoft/tree/main/repo-info) to identify projects and the date since last commit.

**Comment by darko-marinov** [comment]
Does the existing script automatically update `pr-data.csv` to add the notes?

**Comment by KiruthikaJanakiraman** [comment]
No, it does not. I wrote a script to update pr-data.csv, but as mentioned in email, format-checker is failing for the updated csv since I am re-writing the file and the checker runs on all the rows and not just the rows I changed.
I asked Zijie about this and he suggested to update the status manually for now.

File: pr-data.csv [hunk 1]
```diff
@@ -656,11 +656,11 @@ https://github.com/alibaba/druid,ad05a9293b6be2d5562807698b215b673db0417a,create
...
+https://github.com/alibaba/easy-retry,16e0c4156347658c1fd701ad4bf53aeeb214ce50,easy-retry-extensions,com.alibaba.easyretry.extension.mybatis.access.MybatisRetryTaskAccessTest.finish,OD-Vic,,,
+https://github.com/alibaba/easy-retry,16e0c4156347658c1fd701ad4bf53aeeb214ce50,easy-retry-extensions,com.alibaba.easyretry.extension.mybatis.access.MybatisRetryTaskAccessTest.handle,OD-Vic,,,
+https://github.com/alibaba/easy-retry,16e0c4156347658c1fd701ad4bf53aeeb214ce50,easy-retry-extensions,com.alibaba.easyretry.extension.mybatis.access.MybatisRetryTaskAccessTest.stop,OD-Vic,,,
+https://github.com/alibaba/easy-retry,16e0c4156347658c1fd701ad4bf53aeeb214ce50,easy-retry-extensions,com.alibaba.easyretry.extension.mybatis.dao.RetryTaskDAOImplTest.deleteRetryTask,OD-Vic,,,
+https://github.com/alibaba/easy-retry,16e0c4156347658c1fd701ad4bf53aeeb214ce50,easy-retry-extensions,com.alibaba.easyretry.extension.mybatis.dao.RetryTaskDAOImplTest.listRetryTask,OD-Vic,,,
```
Comment (zzjas) [inline pr-data.csv:663]: I think you need to rebase on current main again. These are overwriting new changes merged in IDoFT.

Comment (KiruthikaJanakiraman) [reply pr-data.csv]: Done

**Comment by zzjas** [comment]
Ok to merge

**Pull request merged**